.ci/bump-opbeans-go.yml

```
@@ -0,0 +1,65 @@
+---
+## Workflow to periodically check if there is an available newer APM Go agent version, e.g.
+## "v1.2.3". If so, then update to it.
+name: Bump elastic-apm-go to latest version
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: opbeans-go
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  opbeans-go:
+    kind: github/pullrequest
+    scmid: githubConfig
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+
+sources:
+  elastic-apm-go:
+    name: Get latest elastic-apm-go version
+    kind: shell
+    spec:
+      command: bash .ci/get-latest-version.sh
+      environments:
+        - name: PATH
```

you can get a the GitHub releases directly, you do not need the script
```suggestion
  elastic-apm-go:
    name: Get latest elastic-apm-go version
    kind: githubrelease
    spec:
      owner: "elastic"
      repository: "apm-agent-go"
      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
      username: '{{ requiredEnv "GIT_USER" }}'
      versionFilter:
        kind: latest
    transformers:
      - trimprefix: "v"
```